var/spack/repos/builtin/packages/phist/package.py

```diff
@@ -103,6 +103,10 @@ class Phist(CMakePackage):
 
     # ###################### Patches ##########################
 
+    # the phist repo came with it's own FindMPI.cmake before, which may cause some other
+    # MPI installation to be used than the one spack wants.
+    patch('delete_own_FindMPI.patch', when='@:1.9.6')
```

Comment:
Rather than this patch file, it would probably be easier to add:
```
@when('@:1.9.6')
def patch(self):
    os.unlink('cmake/FindMPI.cmake')
```
(don't forget to add `os` to the imports at the top).